x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/installed_integrations/components/installed_integrations_table.tsx

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import {
+  EuiBasicTable,
+  EuiLink,
+  EuiFlexGroup,
+  EuiFlexItem,
+  type CriteriaWithPagination,
+  EuiToolTip,
+  EuiText,
+  EuiSpacer,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { TableIcon } from '../../../../../../../components/package_icon';
+import type { PackageListItem } from '../../../../../../../../common';
+import { type UrlPagination, useLink, useAuthz } from '../../../../../../../hooks';
+import type { PackageListItemWithExtra } from '../types';
+
+import { InstallationStatus } from './installation_status';
+
+/**
+ * Wrapper to display a tooltip if
```

Comment:
nit: I assume this is an unfinished comment :D 

```suggestion
 * Wrapper to display a tooltip if element is disabled (i.e. due to insufficient permissions)
```